lib/sage_one/request.rb

```diff
@@ -47,6 +48,17 @@ def request(method, path, options)
       end
```

Comment:
add comment: by default auto_traversal is true hence index data will be loaded at once in memory